Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps/Pin.cs

```diff
@@ -16,6 +16,8 @@ public sealed class Pin : BindableObject
 
         public static readonly BindableProperty IsDraggableProperty = BindableProperty.Create("IsDraggable", typeof(bool), typeof(Pin), false);
 
+        public static readonly BindableProperty RotationProperty = BindableProperty.Create("Rotation", typeof(float), typeof(Pin), 0);
```

Comment:
I have error 

>System.TypeInitializationException: The type initializer for 'Xamarin.Forms.GoogleMaps.Pin' threw an exception. ---> System.ArgumentException: Default value did not match return type
Parameter name: defaultValue

Please change last parameter ``0`` to ``0f``.

``BindableProperty.Create("Rotation", typeof(float), typeof(Pin), 0f);``